bitnami/node-exporter/values.yaml

```diff
@@ -0,0 +1,222 @@
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+
+## Additional command line arguments to pass to node-exporter
+extraArgs: {}
+  # collector.filesystem.ignored-mount-points: "^/(dev|proc|sys|var/lib/docker/.+)($|/)"
+  # collector.filesystem.ignored-fs-types: "^(autofs|binfmt_misc|cgroup|configfs|debugfs|devpts|devtmpfs|fusectl|hugetlbfs|mqueue|overlay|proc|procfs|pstore|rpc_pipefs|securityfs|sysfs|tracefs)$"
+
+## Additional volumes to the node-exporter pods
+extraVolumes: []
+#  - name: copy-portal-skins
+#    emptyDir: {}
+
+## Additional volumeMounts to the node-exporter container
+extraVolumeMounts: []
+#  - name: copy-portal-skins
+#   mountPath: /var/lib/lemonldap-ng/portal/skins
+
+## Service account for Node Exporter to use.
+## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/
+##
+serviceAccount:
+  ## Specifies whether a ServiceAccount should be created
+  ##
+  create: true
+  ## The name of the ServiceAccount to use.
+  ## If not set and create is true, a name is generated using the node-exporter.fullname template
+  # name:
+
+## SecurityContext configuration
+##
+securityContext:
+  enabled: true
+  runAsUser: 1001
+  fsGroup: 1001
+
+## Node Exporter Service
+##
+service:
+  ## Kubernetes service type and port number
+  ##
+  type: ClusterIP
+  port: 9100
+  # clusterIP: None
+
+  ## Specify the nodePort value for the LoadBalancer and NodePort service types.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+  ##
+  # nodePort: 30080
+
+  ## Set the LoadBalancer service type to internal only.
+  ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
+  ##
+  # loadBalancerIP:
+
+  ## Load Balancer sources
+  ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+  ##
+  # loadBalancerSourceRanges:
+  # - 10.10.10.0/24
+
+  ## Provide any additional service annotations
+  ##
+  annotations: {}
+
+  ## Provide any additional service labels
+  ##
+  labels: {}
+
+# The update strategy to apply to the DaemonSet
+##
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxUnavailable: 1
+
+# Expose the service to the host network
+hostNetwork: true
+
+# minReadySeconds to avoid killing pods before we are ready
+##
+minReadySeconds: 0
+
+## Priority class assigned to the Pods
+##
+priorityClassName: ""
+
+## Configure resource requests and limits
+## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+##
+resources: {}
```

Comment:
Use the format below for resources:

- https://github.com/bitnami/charts/blob/master/bitnami/nginx/values.yaml#L94